src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -174,16 +190,26 @@ function newRepeatableElement(container, field_group, values) {
             var field_name = container.data('repeatable-identifier');
             var new_field_group = field_group.clone();
 
-            //this is the container for this repeatable group that holds it inside the main form.
+            // this is the container that holds the group of fields inside the main form.
             var container_holder = $('[data-repeatable-holder='+field_name+']');
 
```

Comment:
```suggestion
            var add_button = container_holder.closest('.form-group').find('.add-repeatable-element-button');
```
There are multiple calls to this button, I think it's a good idea to store it as a var.